### ISSUE TITLE
iOS: mailto subject fails to decode

### ISSUE THREAD

**Issue by Krinkle** [Ranchero-Software/NetNewsWire#4099]
* app: NetNewsWire 6.1.4 (Build 6120).
* platform: iOS 16.6.
* account: Feedbin.

When following a `mailto` link, the subject fails to decode. I'm guessing it's either interpreted too directly, or double encoded at some point. Clicking the same link from a web page in a browser on the same device works as expected. E.g. in Mobile Safari, or when browsing the article web page within a WebView in the NetNewsWire app, or e.g. a wrapper for Mobile Safari like Firefox on iOS.

| Nay | Yay |
|--|--
| ![Subject: RE%3A%20An%20Internet](https://github.com/Ranchero-Software/NetNewsWire/assets/156867/7eb7c80d-abd8-48aa-9364-d1892fab9b7b) | ![Subject: RE: An Internet](https://github.com/Ranchero-Software/NetNewsWire/assets/156867/3677b1dc-d737-4eea-8742-3082b900728b)

Example: 

https://timotijhof.net/posts/2023/an-internet-of-php/
```html
<a target="_blank" href="mailto:hello@timotijhof.net?subject=RE%3A%20An%20Internet%20of%20PHP">Reply by email</a>
```

The corresponding entry in the RSS feed for https://timotijhof.net/ (<https://timotijhof.net/feed/>):
```xml
<generator>https://wordpress.org/?v=6.3.1</generator>
<item>
	<title>An Internet of PHP</title>
	<link>https://timotijhof.net/posts/2023/an-internet-of-php/</link>
	…
	<content:encoded><![CDATA[
<p>PHP is <strong>big</strong>. …</p>
…
<hr/><p><a target="_blank" href="mailto:hello@timotijhof.net?subject=RE%3A%20An%20Internet%20of%20PHP">Reply via email</a></p>]]></content:encoded>
```

In case it makes a difference, I'm subscribed via Feedbin.

* Similar issue from 2021 (closed). https://github.com/Ranchero-Software/NetNewsWire/issues/3281, https://github.com/Ranchero-Software/NetNewsWire/issues/2711
* Related issue from 2020 (closed). https://github.com/Ranchero-Software/NetNewsWire/issues/2256

**Comment by GetToSet** [comment]
A fix to this issue has already been landed on the `main` branch (ce6d114, #3719).

**Comment by pimoore** [comment]
@brentsimmons It seems as though this issue has resurfaced in the current version of NNW on iOS. Even though a mailto link including subject was set to be encoded within the feed itself, the characters are still not decoding properly in the email. Checked in both Feedbin and miniflux with the same feed, and the issue persists. Definitely isn't something with the link itself, as it works properly in Reeder on iOS/iPadOS (I don't have a Mac to verify there).
